### DISCUSSION TITLE
POST api/v1/statuses returning 404, how?

### DISCUSSION THREAD

**Discussion by Slugger** [mastodon/mastodon#24621] [Q&A]
I have a bot running that periodically posts statuses by calling `POST api/v1/statuses`.  90% of the time, things work as expected, but sometimes this call fails with a 404 error: `Record not found`.  404 is not a documented return code for this api call.  So I'm not sure what's causing the 404.  This is on a `POST`, so creating a new status.

Only certain attempts at this api call fail while others post just fine.  Looking at the payload of the failed vs. successful posts, I'm not seeing anything obvious.  What would cause this call to return a 404?

**Comment by mjankowski** [comment]
Did you ever work this out?

Was this happening only on certain servers, or everywhere?

**Reply by Slugger** [reply]
Nah.  I was only running it on one server so not sure if it was server specific.  That was 2.5 years ago and I've long since ended that side project.  The bot hasn't been running in well over a year now.
